cmd/podman/pod_rm.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	podRmFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Force removal of a running pod.  The default is false",
+		},
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "Remove all pods",
+		},
+		cli.BoolFlag{
+			Name:  "rmctr, r",
+			Usage: "Remove all containers in a pod",
```

Comment:
This option is not about *pod* removal.  I think you want a command to remove containers from a pod (the inverse of creating a new container within the pods, which you don't seem to support yet).  Perhaps that's just `podman delete CONTAINER ...`, in which case you'd want to support `podman delete --pod=POD` to delete all the (stopped?) containers from *POD*.

Reply:
I think this more deserves to be `--force` than `--r` -- it's forcibly removing the pod, including any containers that may be in it.

Reply:
Basically, I think we should fold the existing `force` flag into here, and have one flag to do both (specify `--force` and the pod is guaranteed to be gone).

Reply:
I just used the variable names in the libpod api because, quite frankly, the usage confused me at first. Upon reflection, the current -f option is more of a --stop (stop running containers). However, I can't see how this can be useful in the rm command without --rmctr. --rmctr makes sense as -r to me because it recursively removes the contents of the pod (containers). I like the flags as they are now because rm -rf aptly describes the situation compared to bash rm -rf. But realistically, the explaination is too difficult to get through. Here are my thoughts:
--force option -> --stop (if we have it at all)
-r stays the same (maybe becomes --rm-ctr)
-f becomes --force && --rmctr as it stands (stop all containers and force the removal)

I can also imagine a --detach option that detaches the containers from the pod before removing the pod, to be able to keep containers but still get rid of the pod. 

WDYT @mheon @wking

Reply:
ok yeah I agree with that